tests/test_agg.py

```diff
@@ -308,22 +308,33 @@ def test_aggs(self):
         self.assertEqual(list(iter(a)), a.buckets)
         self.assertEqual(a.buckets[0].key, 0)
         self.assertEqual(a.buckets[0].doc_count, 7353499)
+        self.assertEqual(str(a.buckets[0]), '<Bucket key=0 doc_count=7353499>')
```

Comment:
You have implemented `__repr__` so I'd rather used `repr(a.buckets[0])`, because str will look for `__str__` 
implementation and than fallback to `__repr__` so if someone implement `__str__` these tests will fail.